app/src/main/dist/config/crate.yml

```diff
@@ -79,47 +79,26 @@
 
 # To allow authenticated access to CrateDB from specific hosts, you must
 # enable the host based authentication setting.
-#auth.host_based.enabled: true
+# The default value is `false`.
+auth.host_based.enabled: true
 
 # Client access and authentication is then configured via the host based
-# config, a map of remote client access entries. For example:
-#auth:
-#  host_based:
-#    enabled: true
-#    config:
-#      0:
-#        user: mike
-#        address: 32.0.0.0/8
-#        method: trust
-#        protocol: pg
-#      a:
-#        user: barb
-#        address: 172.16.0.0
-#        protocol: pg
-#      b:
-#        user: crate
-#        address: 32.0.0.0/8
-#        method: trust
-#      y:
-#        user: eleven
-#        protocol: pg
-#      e:
-#        user: dustin
-#        address: 172.16.0.0
-#        method: trust
-#        protocol: http
-#      z:
-#        method: trust
-
-# When CrateDB is started, the cluster contains one predefined superuser,
-# called `crate`. To enable trust based authentication for the super user,
-# `crate` must be specified in the host_based setting, as follows:
-#auth:
-#  host_based:
-#    enabled: true
-#    config:
-#      0:
-#        user: crate
+# config, a map of remote client access entries.
+# The following example is a sane configuration that covers a common use case:
+# * The predefined superuser `crate` has trusted access from localhost.
+# * All other users require username/password authentication from anywhere.
+#   Note that this authentication method is only available via Postgres
+#   Protocol, meaning that with this configuration, connecting from non-local
+#   hosts via HTTP would not be possible!
+auth:
+  host_based:
+    config:
+      0:
+        user: crate
+        address: 127.0.0.1
```

Comment:
this does not work with local IPv6 address (`::1`)
as we do not have the possibility to specify multiple addresses we would need to add another entry

```yaml
1:
  user: crate
  address: ::1
  method: trust
```

Reply:
https://github.com/crate/crate/pull/6541

Reply:
will change the `address` to `_local_` once the PR above is merged into master